test/README.md

```diff
@@ -18,8 +18,8 @@ It also tests usage with `bitcoind`.
 It uses the [Coldcard simulator](https://github.com/Coldcard/firmware/tree/master/unix#coldcard-desktop-simulator).
 It also tests usage with `bitcoind`.
 
-`setup_environment.sh` will build the Trezor emulator, the Coldcard simulator, the Keepkey emulator, the Digital Bitbox simulator, and `bitcoind`.
-if run in the `test/` directory, these will be built in `work/test/trezor-mcu`, `work/test/firmware`, `work/test/keepkey-firmware`, `work/test/mcu`, and `work/test/bitcoin` respectively.
+There is a script to build the Trezor emulator, the Coldcard simulator, the Keepkey emulator, the Digital Bitbox simulator, and `bitcoind`. On Ubuntu / Debian first install: `sudo apt-get install libsdl2-dev`. On all systems: `pip install pipenv`
```

Comment:
We already list the specific dependencies below. I think it would be better to just point people down there instead of trying to list them here.